website/server/controllers/api-v3/members.js

```diff
@@ -519,9 +647,9 @@ api.sendPrivateMessage = {
  * @apiName TransferGems
  * @apiGroup Member
  *
- * @apiParam (Body) {String} message The message
```

Comment:
why were all the (Body), (Path) and (Query) removed?

Reply:
@paglias the example API didn't have these, so I assumed they were taken out for a reason. 
https://github.com/HabitRPG/habitica/blob/ee15e29ba4d34812c4fd4e5836dc310d713a7db9/website/server/controllers/api-v3/groups.js#L643-L683

I don't mind putting them back in if that's what we want to do, though.

Reply:
We have them throughout the rest of the documentation (see e.g. all the `@apiParam` entries in `website/server/controllers/api-v3/tasks.js`), so I'd say it's better to have them.

Reply:
@paglias @SabreCat Fixed!